package.json

```diff
@@ -325,18 +325,18 @@
           "name": "Help",
           "type": "webview",
           "when": "malloy.extensionStatus == installed || resourceFilename =~ /\\.malloy$/"
+        },
+        {
+          "id": "malloyConnections",
+          "name": "Malloy Connections",
+          "when": "malloy.webviewPanelFocused == true || resourceFilename =~ /\\.malloy(nb)?$/"
```

Comment:
It seems like the two panels in this section should have the same `when`.

Reply:
Made the change to make the two panels having the same `when`.